sdk/typescript/src/client/types/params.ts

```diff
@@ -278,7 +278,7 @@ export interface GetEpochsParams {
 /** Address related metrics. Exclusively served by the indexer. */
 export interface GetLatestAddressMetricsParams {}
 /**
- * Return the latest IOTA system state object on networks supporting protocol version `< 4`. These are
+ * Return the latest IOTA system state object on networks supporting protocol version `< 5`. These are
```

Comment:
this change is unrelated to the PR, not sure how we slipped when merging PCR2 but fyi the change is included here https://github.com/iotaledger/iota/pull/5801, so this can be reverted